cmd/system-probe/subcommands/run/command.go

```diff
@@ -111,6 +116,17 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 				workloadmetafx.Module(workloadmeta.Params{
 					AgentType: workloadmeta.Remote,
 				}),
+				// Provide tagger module
+				remoteTaggerFx.Module(tagger.RemoteParams{
+					RemoteTarget: func(c config.Component) (string, error) { return fmt.Sprintf(":%v", c.GetInt("cmd_port")), nil },
+					RemoteTokenFetcher: func(c config.Component) func() (string, error) {
+						return func() (string, error) { return security.FetchAuthToken(c) }
+					},
+					RemoteFilter: taggerTypes.NewMatchAllFilter(),
+				}),
+				fx.Invoke(func(wmeta workloadmeta.Component, tagger tagger.Component) {
+					proccontainers.InitSharedContainerProvider(wmeta, tagger)
```

Comment:
Why do we need this ? I believe this is the reason behind the explosion of windows dependencies, and AFAICS it's not really needed to have a working workloadmeta or tagger

Tested in https://github.com/DataDog/datadog-agent/pull/31444 for reference